plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeAwsConnectorSmokeTest.java

```diff
@@ -26,23 +25,20 @@
 public abstract class BaseDeltaLakeAwsConnectorSmokeTest
         extends BaseDeltaLakeConnectorSmokeTest
 {
-    protected DockerizedMinioDataLake dockerizedMinioDataLake;
+    protected HiveMinioDataLake hiveMinioDataLake;
 
     @Override
-    DockerizedDataLake createDockerizedDataLake()
+    HiveMinioDataLake createHiveMinioDataLake()
     {
-        dockerizedMinioDataLake = new DockerizedMinioDataLake(
-                bucketName,
-                getHadoopBaseImage(),
-                ImmutableMap.of("io/trino/plugin/deltalake/core-site.xml", "/etc/hadoop/conf/core-site.xml"),
-                ImmutableMap.of());
-        return dockerizedMinioDataLake;
+        hiveMinioDataLake = new HiveMinioDataLake(bucketName, ImmutableMap.of(), "ghcr.io/trinodb/testing/hdp2.6-hive");
```

Comment:
Pls avoid using magic strings for the docker image - Let's introduce a constant for this purpose.

Reply:
NIT: I remember there is a class where we store docker image version. 
`io.trino.plugin.deltalake.util.DockerImages`
Maybe let's extract it to test containers and keep those values there